src/components/SwipePan/index.tsx

```diff
@@ -48,6 +50,7 @@ const SwipePan = ({
     onActive: (event, ctx) => {
       x.value = ctx.startX + event.translationX;
       y.value = ctx.startY + event.translationY;
+      runOnJS(onActive)(x, y);
```

Comment:
Have you test onActive callback for performance issues? It's will generate a lot of call per second, at least we need to warn of a possible performance issue in the docs for this onSwipeActive method. Also would be great if you investigate some solutions to improve this.

Reply:
I updated this part. The new solution removes all unnecessary function calls. Check it